src/components/Chat/Chats.js

```diff
@@ -20,7 +20,7 @@ const Chats = ({ setChat }) => {
     <div className='chats'>
       { conversation.length > 0 && conversation.map((chat, i) => (
         <div className='user-chat' key={i} onClick={() => { handleSelect(chat); setChat() }}>
-          {chat.Users.photoURL && <img className='user-picture' src={chat.Users.photoURL }/>}
+          {chat.Users.photo && <img className='user-picture' src={`${process.env.REACT_APP_SPACES_URL}${chat.Users.photo}`}/>}
```

Comment:
You shouldn't build the image urls on the frontend. Instead generate the full url on the backend.